### PR TITLE
Security/upgrade node-notifier

### PR DESCRIPTION
**Pull request by ethan605**
# Problem

Ref: https://github.com/advisories/GHSA-5fw9-fq32-wv5p

# Solution

Upgrade to `jest@26.6.3` and `jest-cli@26.6.3` to have latest `node-notifier@8.0.1`

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [n/a] Has the CHANGELOG been updated?
- [n/a] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [n/a] Have any new automated tests been implemented or the existing ones changed?
- [n/a] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1329-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1329-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1329-pr-onfido-sdk-ui-onfido.surge.sh

File: licenses.json [hunk 1]
```diff
@@ -9491,7 +9163,7 @@
     "publisher": "Web SDK Customer Support",
     "email": "web-sdk@onfido.com",
     "url": "https://github.com/onfido",
-    "path": "onfido-sdk-ui",
+    "path": "/home/ethanify/Work/onfido-sdk-ui",
```
Comment (kopijunkie) [inline licenses.json:9166]: We should just keep this as "onfido-sdk-ui" from my discussion with @stefaniacardenas when reviewing this licenses.json file generator's implementation PR.

**Review by kopijunkie — COMMENTED** [review]
Shouldn't dependabot have raised a PR for this? 🤔

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1329-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1329-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1329-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by ethan605** [comment]
> Shouldn't dependabot have raised a PR for this?

I manually triggered the security update button but nothing created. Probably because `node-notifier` is not explicitly installed.

**Review by kopijunkie — APPROVED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1329-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1329-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1329-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**